packages/client/src/types.ts

```diff
@@ -102,6 +102,7 @@ export const fileNameSymbol = Symbol('fileName');
 export interface VercelConfig {
   [fileNameSymbol]?: string;
   name?: string;
+  meta?: string;
```

Comment:
I think meta should be `Dictionary<string>` or `string[]`

Reply:
Done, Thank You. Opt for `string[]`